yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/server/SpecialCodeVersion.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * SpecialCodeVersion.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2025 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.relational.yamltests.server;
+
+import com.apple.foundationdb.relational.yamltests.block.FileOptions;
+
+import javax.annotation.Nonnull;
+import java.util.EnumMap;
+import java.util.Objects;
+
+public class SpecialCodeVersion implements CodeVersion {
+    public enum SpecialCodeVersionType {
+        MIN("!min_version", false),
+        CURRENT(FileOptions.CurrentVersion.TEXT, true),
```

Comment:
I _feel_ like this should provide what the text for CurrentVersion is, rather than depending on `FileOptions`